packages/debug/src/browser/debug-session.tsx

```diff
@@ -759,4 +741,27 @@ export class DebugSession implements CompositeTreeElement {
         return this.threads;
     }
 
+    protected handleContinued = ({ body: { allThreadsContinued, threadId } }: DebugProtocol.ContinuedEvent): void => {
+        if (allThreadsContinued !== false) {
+            this.clearThreads();
+        } else {
+            this.clearThread(threadId);
+        }
+    };
+
+    protected handleStopped = async ({ body }: DebugProtocol.StoppedEvent): Promise<void> => {
```

Comment:
Why not methods? Properites are not overridable, only reassignable. Such pattern should be only used in react component to avoid breaking caches. In other places it looks alient.

Reply:
I used this pattern to avoid binding issues, but I'd be fine with other patterns that achieve the same effect. In this context, would you prefer `someMethod.bind(this)` or `(e) => this.someMethod(e)`?

Reply:
both are fine with me, most important that here we should methods, not properties, otherwise you cannot override them